Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -377,8 +373,8 @@ private void animateBottomSheetClosedAndFinishDropInWithResult(DropInResult drop
             // when the bottom sheet transitions to the "HIDDEN" state; the activity will finish
             dropInViewModel.setBottomSheetState(BottomSheetState.HIDE_REQUESTED);
         } else {
-            // no need to animate bottom sheet hidden; finish activity with default Android animation
-            finishDropInWithPendingResult(false);
+            // no need to animate activity hidden since bottom sheet is not visible
+            finishDropInWithPendingResult(DropInExitTransition.NO_ANIMATION);
```

Comment:
I'm a little confused why this would ever be a scenario? If we're delivering a DropInResult, wouldn't the sheet always be presented? I'm a little rusty here and would appreciate some additional context I am probably missing!

Reply:
Yeah that's a good call. `AddCardFragment` has it's own loader when the user submits their card information, so we don't animate when dismissing the activity. I wonder now though if `DropInExitTransition.DEFAULT` would make more sense here since no animation after tokenizing a card might be too abrupt.

Reply:
Ah gotcha - that makes more sense to me!